### DISCUSSION TITLE
Custom View - complexity

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17155] [Feature improvement]
Was wondering, (as per below short movie of one of my filter) if it would be possible to include certain conditions in one field divided by some spec character? 
As you see filter contains large number of conditions (to filter out some certain English words etc) which I need to reproduce in another very similar module. 

Best!

![filter](https://github.com/YetiForceCompany/YetiForceCRM/assets/84110093/c1e2cd62-37fc-44c6-819e-09a86554a297)

**Comment by ArniSamano** [comment]
Someone have similar thing: https://github.com/YetiForceCompany/YetiForceCRM/discussions/17018
